.github/workflows/ci.yaml

```diff
@@ -110,7 +110,7 @@ jobs:
           cd index/server
           export GOPATH=$(go env GOPATH)
           go install github.com/deepmap/oapi-codegen/cmd/oapi-codegen@${{ env.OAPI_CODEGEN_VERSION }}
-          bash ./build.sh
+          bash ./build.sh && bash ./build.sh linux/arm64
```

Comment:
Since this line is just testing to make sure the builds are working on a PR I can split it up so instead of them being combined they can each have their own setup? One for amd64 and one for arm64?

On the other hand with logging I can also try to alter the script to add some more logging into it, I will try both and see which one is more clear! I don't mind checking it out in this PR as I feel proper logging for the arch is part of the scope for this.